tests/test_dcos_e2e/docker_utils/test_loopback.py

```diff
@@ -0,0 +1,69 @@
+from typing import Tuple
+
+import docker
+import pytest
+
+from dcos_e2e.docker_utils import DockerLoopbackVolume
+
+# We ignore this error because it conflicts with `pytest` standard usage.
+# pylint: disable=redefined-outer-name
+
+
+@pytest.fixture
+def loopback_sidecar() -> Tuple[docker.models.containers.Container, str]:
+    return DockerLoopbackVolume.create(size=1)
+
+
+class TestDockerLoopbackVolume:
+    """
+    Tests for setting device mapping on master or agent Docker containers.
+    """
+
+    def test_loopback(self) -> None:
+        """
+        An instance of `DockerLoopbackVolume` provides a block device.
+        """
+        labels = {'foo': 'bar'}
+
+        with DockerLoopbackVolume(size=1, labels=labels) as device:
+            cmd = 'lsblk {}'.format(device.path)
+            exit_code, output = device.container.exec_run(cmd=cmd)
+            assert exit_code == 0, cmd + ': ' + output.decode()
+
+            for key, value in labels.items():
+                assert device.container.labels[key] == value
+
+    def test_create(self) -> None:
+        """
+        Calling `DockerLoopbackVolume.create` creates a sidecar container.
+        """
+        labels = {'foo': 'bar'}
+
+        container, path = DockerLoopbackVolume.create(
+            size=1,
+            labels=labels,
+        )
+
+        try:
+            for key, value in labels.items():
+                assert container.labels[key] == value
+
+            cmd = 'lsblk {}'.format(path)
+            exit_code, output = container.exec_run(cmd=cmd)
+            assert exit_code == 0, cmd + ': ' + output.decode()
+
+        finally:
+            DockerLoopbackVolume.destroy(container)
+            container.stop()
+            container.remove(v=True)
+
+    def test_destroy(
+        self,
+        loopback_sidecar: Tuple[docker.models.containers.Container, str],
+    ) -> None:
+        """
+        Calling `DockerLoopbackVolume.destroy` destroys a sidecar container.
+        """
+        DockerLoopbackVolume.destroy(loopback_sidecar[0])
```

Comment:
As a reader of this test it is not clear to me how this demonstrates what the test docstring says.

Reply:
@nfnt Any chance you can address this?

Reply:
I've updated the docstring to clarify that in case of a failure an expection would be thrown.